### PR TITLE
fix issue(#47976)Invalid value error when creating service from expor…

### PR DESCRIPTION
**Pull request by zjj2wry**
…ted config

**What this PR does / why we need it**:
close issue #47976
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/registry/core/service/strategy.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func (svcStrategy) Export(ctx genericapirequest.Context, obj runtime.Object, exa
 		return nil
 	}
 	if t.Spec.ClusterIP != api.ClusterIPNone {
-		t.Spec.ClusterIP = "<unknown>"
+		t.Spec.ClusterIP = ""
```
Comment (dixudx) [inline pkg/registry/core/service/strategy.go]: Please also fix `strategy_test.go`.

Comment (zjj2wry) [reply pkg/registry/core/service/strategy.go]: updated

**Comment by cmluciano** [comment]
xref: #28551 #28548

**Comment by cmluciano** [comment]
/retest

cluster setup flake

**Comment by caseydavenport** [comment]
Should this be in the v1.8 milestone since it fixes https://github.com/kubernetes/kubernetes/issues/47976?

**Review by mkumatag — APPROVED** [review]

**Comment by dims** [comment]
/kind bug

(as the issue referenced is a bug)

**Comment by mengqiy** [comment]
/assign @liggitt @smarterclayton 

Can someone approve this?

**Comment by smarterclayton** [comment]
Thanks for fixing

**Pull request merged**